ci-operator/step-registry/acm/tests/grc/acm-tests-grc-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# The variables defined in this step come from files in the `SHARED_DIR` and credentials from Vault.
+# SECRETS_DIR="/tmp/secrets"
+
```

Comment:
We discussed yesterday storing the data about the managed clusters into the $SHARED_DIR.

I'm not sure if you need to set an env var using these values here. Something like
```
MANAGED_CLUSTER_NAME=$(cat $SHARED_DIR/managed.cluster.name)
export MANAGED_CLUSTER_NAME
MANAGED_CLUSTER_NAME=$(cat $SHARED_DIR/managed.cluster.base.domain)
export MANAGED_CLUSTER_BASE_DOMAIN
MANAGED_CLUSTER_NAME=$(cat $SHARED_DIR/managed.cluster.api.url)
export MANAGED_CLUSTER_API_URL
MANAGED_CLUSTER_NAME=$(cat $SHARED_DIR/managed.cluster.username)
export MANAGED_CLUSTER_USERNAME
MANAGED_CLUSTER_NAME=$(cat $SHARED_DIR/managed.cluster.password)
export MANAGED_CLUSTER_PASSWORD
```

Reply:
GRC could be run without the managed cluster, as the tests will be querying it.
As for OBS for example, the managed cluster info is required, and this info is not provided, the test suite would fail